mingw-w64-pico-sdk/PKGBUILD

```
@@ -0,0 +1,40 @@
+# Contributer: thewh1teagle <61390950+thewh1teagle@users.noreply.github.com>
+
+_realname=pico-sdk
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=1.5.1
+pkgrel=1
+pkgdesc="Libraries for C/C++ development on RP2040 microcontrollers. (mingw-w64)"
+arch=("any")
+mingw_arch=("mingw64" "ucrt64" "clang64")
+url="https://github.com/raspberrypi/pico-sdk"
+license=("spdx:BSD-3-Clause")
+makedepends=("git")
+options=("!strip")
+install=${_realname}.install
+source=("git+${url}.git#tag=${pkgver}")
+sha256sums=("SKIP")
+
+prepare() {
+    cd "${srcdir}/${_realname}"
+    git submodule update --init --depth=1
+}
+
+package() {
+    # Install docs
+    install -Dm644 "${srcdir}/${_realname}/README.md" -t "${pkgdir}/share/doc/${pkgname}"
```

```suggestion
    install -Dm644 "${srcdir}/${_realname}/README.md" -t "${pkgdir}${MINGW_PREFIX}/share/doc/${pkgname}"
```
replace /usr with ${MINGW_PREFIX}